mods/beds/functions.lua

```diff
@@ -137,6 +138,9 @@ function beds.on_rightclick(pos, player)
 	local ppos = player:getpos()
 	local tod = minetest.get_timeofday()
 
+	beds.set_spawn(name)
```

Comment:
unrelated functionality change! please remove and submit a *separate* pr for this change.